app/helpers/admin_helper.rb

```diff
@@ -6,7 +6,7 @@ def admin_nav_to(name, path)
                             [path, nil]
                           end
 
-    link = link_to_accessible name, link_path, class: "govuk-link govuk-link--no-visited-state"
+    link = govuk_link_to name, link_path, class: "govuk-link govuk-link--no-visited-state"
```

Comment:
I think the class `govuk-link` can be removed as it is applied by default by the helper `govuk_link_to`

Also, the visited class should be set using the [helpers option](https://govuk-components.netlify.app/helpers/link/#other-link-styles) `no_visited_state`

In short, 
`link = govuk_link_to name, link_path, class: "govuk-link govuk-link--no-visited-state"`

becomes 

`link = govuk_link_to name, link_path, { no_visited_state: true }`

Reply:
Thank you for the tip. I have made these suggested changes now.